website/src/pages/v1/compose/source-handlers/json-schema.mdx

```diff
@@ -8,7 +8,10 @@ import { Callout } from '@theguild/components'
 
 # JSON Schema or Samples
 
-![image](https://user-images.githubusercontent.com/20847995/79218994-11434880-7e5a-11ea-8594-08185526080f.png)
```

Comment:
Did you remove these images on purpose? 😅

Reply:
they don't add anything in terms of aesthetics and functionality and look out of place in these places